lib/mix/lib/mix/dep.ex

```diff
@@ -167,10 +167,10 @@ defmodule Mix.Dep do
     do: "lock mismatch: the dependency is out of date (run \"mix deps.get\" to fetch locked version)"
```

Comment:
perfect. what's is consufing is reading an "out of date message" when current (local) dependency version is 0.2.0, and version in mix.lock  is 0.1.0.